Makefile

```diff
@@ -532,9 +486,11 @@ $(eval $(call assert_boolean,WARMBOOT_ENABLE_DCACHE_EARLY))
 $(eval $(call assert_numeric,ARM_ARCH_MAJOR))
 $(eval $(call assert_numeric,ARM_ARCH_MINOR))
 
+ifeq ($(ARM_ARCH_MAJOR),7)
```

Comment:
The below `assert_boolean` s can be moved to armv7-a-cpus.mk as well.

Reply:
ok